### PR TITLE
Renamed config variables. Fixed error in settings.

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: openslides/core/signals.py [hunk 1]
```diff
@@ -40,47 +40,48 @@ def setup_general_config(sender, **kwargs):
...
         default_value='',
         form_field=forms.CharField(
             widget=forms.TextInput(),
             label=ugettext_lazy('Event organizer'),
             required=False))
 
-    projector_enable_logo = ConfigVariable(
-        name='projector_enable_logo',
+    general_projector_enable_logo = ConfigVariable(
+        name='general_projector_enable_logo',
```
Comment (emanuelschuetze) [inline openslides/core/signals.py]: I would prefer that we use a own prefix for all "projector" configs to group projector configs in one section. So my proposal: remove "general_" from all projector configs.

File: openslides/core/signals.py [hunk 2]
```diff
@@ -106,27 +107,8 @@ def setup_general_config(sender, **kwargs):
...
-        name='welcome_title',
-        default_value=_('Welcome to OpenSlides'),
-        translatable=True,
-        form_field=forms.CharField(
-            widget=forms.TextInput(),
-            label=ugettext_lazy('Title'),
-            help_text=ugettext_lazy('Also used for the default welcome slide.'),
-            required=False))
-
-    welcome_text = ConfigVariable(
```
Comment (emanuelschuetze) [inline openslides/core/signals.py]: Why welcome_title and welcome_text is removed? We use the title also for the welcome slide.

Comment (normanjaeckel) [inline openslides/core/signals.py]: Reverted. Config vars now belong to projector.

File: openslides/core/signals.py (hunk 1 above)
Comment (normanjaeckel) [inline openslides/core/signals.py]: Done.

**Comment by normanjaeckel** [comment]
All done. Ready for review.

**Comment by emanuelschuetze** [comment]
Ok.

File: openslides/assignments/models.py [hunk 3]
```diff
@@ -241,12 +241,12 @@ def create_poll(self):
...
         # Find out the method of the election
-        if config['assignment_poll_vote_values'] == 'votes':
+        if config['assignments_poll_vote_values'] == 'votes':
             yesnoabstain = False
-        elif config['assignment_poll_vote_values'] == 'yesnoabstain':
+        elif config['assignments_poll_vote_values'] == 'yesnoabstain':
             yesnoabstain = True
         else:
-            # config['assignment_poll_vote_values'] == 'auto'
+            # config['assignments_poll_vote_values'] == 'auto'
```
Comment (ostcar) [inline openslides/assignments/models.py]: Could you remove this line?

**Comment by ostcar** [comment]
One (optional) remark.
 
Please rebase and merge

File: openslides/assignments/models.py (hunk 3 above)
Comment (normanjaeckel) [inline openslides/assignments/models.py]: This line is for readability so you can easily see that there are 3 cases and which cases there are.

**Pull request merged**